templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -765,27 +778,91 @@ Resources:
+                  - es:DescribeElasticsearchDomains
+                  - es:ESHttpPut
+                  - es:CancelElasticsearchServiceSoftwareUpdate
+                  - es:DescribeReservedElasticsearchInstances
+                  - es:DescribeOutboundConnections
+                  - es:ESHttpPatch
+                  - es:DeleteInboundConnection
+                  - es:DeleteInboundCrossClusterSearchConnection
+                  - es:UpdateElasticsearchDomainConfig
+                  - es:PurchaseReservedInstanceOffering
+                  - es:CreateServiceRole
+                  - es:DescribeReservedInstances
+                  - es:AddTags
+                  - es:DeleteElasticsearchDomain
+                  - es:DescribePackages
+                  - es:DescribeElasticsearchInstanceTypeLimits
+                  - es:GetCompatibleVersions
+                  - es:AcceptInboundCrossClusterSearchConnection
+                  - es:ListElasticsearchInstanceTypes
+                  - es:DescribeOutboundCrossClusterSearchConnections
+                  - es:DescribeDomain
+                  - es:ListTags
+                  - es:CreateElasticsearchServiceRole
+                  - es:DeleteDataPrepperPipeline
+                  - es:GetUpgradeHistory
+                  - es:DescribeReservedInstanceOfferings
+                  - es:ESHttpPost
+                  - es:CreateOutboundConnection
+                  - es:CreateOutboundCrossClusterSearchConnection
+                  - es:DissociatePackage
+                  - es:ESHttpHead
+                  - es:CreateDataPrepperPipeline
+                  - es:DescribeDataPrepperPipeline
+                  - es:ListDomainsForPackage
+                  - es:ESHttpDelete
+                  - es:DeletePackage
+                  - es:CreateDomain
+                  - es:DeleteOutboundConnection
+                  - es:CreateElasticsearchDomain
+                  - es:DescribeInboundCrossClusterSearchConnections
+                  - es:DescribeInboundConnections
+                  - es:ListPackagesForDomain
+                  - es:PurchaseReservedElasticsearchInstanceOffering
+                  - es:GetPackageVersionHistory
+                  - es:DeleteOutboundCrossClusterSearchConnection
+                  - es:GetCompatibleElasticsearchVersions
+                  - es:AcceptInboundConnection
+                  - es:UpdateDataPrepperPipeline
+                  - es:RejectInboundConnection
+                  - es:RejectInboundCrossClusterSearchConnection
+                  - es:ESCrossClusterGet
+                  - es:UpdateDomainConfig
+                  - es:StartElasticsearchServiceSoftwareUpdate
+                  - es:DescribeDomains
+                  - es:ListInstanceTypeDetails
+                  - es:RemoveTags
+                  - es:ListElasticsearchVersions
+                  - es:AssociatePackage
+                  - es:DescribeElasticsearchDomain
```

Comment:
Going to assume our CI plans are all passing with this?

Reply:
Yes